charts/hocs-case-creator/values.yaml

```diff
@@ -26,3 +23,9 @@ hocs-generic-service:
       caseworkService: https://hocs-casework.{{ .Release.Namespace }}.svc.cluster.local
       workflowService: https://hocs-workflow.{{ .Release.Namespace }}.svc.cluster.local
       ignoreMessages: 'false'
+
+jobs:
+  process:
+    enabled: false
+    maxMessages: 100
```

Comment:
This is a arbitrary number for now until one is fully decided, could be more suitable as `10` for now.